packages/netdog/netdog.spec

```diff
@@ -90,7 +90,7 @@ install -p -m 0755 ${HOME}/.cache/wicked/%{__cargo_target}/release/netdog %{buil
 %endif
 
 %post -n %{_cross_os}netdog-wicked -p <lua>
-fs.link("%{_cross_bindir}/netdog-wicked", "%{_cross_bindir}/netdog")
+posix.link("%{_cross_bindir}/netdog-wicked", "%{_cross_bindir}/netdog")
```

Comment:
stylistic nit that might appear strangely in the GitHub UI. I'd suggest rebasing this change to the one that introduces the new RPM, since it's otherwise irrelevant here.